packages/config-loader/package.json

```diff
@@ -43,15 +43,17 @@
     "json-schema-traverse": "^1.0.0",
     "typescript-json-schema": "^0.51.0",
     "yaml": "^1.9.2",
-    "yup": "^0.32.9"
+    "yup": "^0.32.9",
+    "node-fetch": "2.6.5"
```

Comment:
@Rugvip  A bit late, but I think we should prefer to use `cross-fetch` here, like we do elsewhere. Maybe the fixed version could also become a problem.

Reply:
@Fox32 I didn't see this comment until now. For posterity, there was a discussion about what to do about this, and the consensus was to actually prefer to use node-fetch instead on backend-specific packages since its types align with node needs and is a better fit. I thought I might spend some time to do that rewrite across the board where possible. Does that sound sensible?

Reply:
@freben yeah, I followed that discussion 👍

For reference: #8127